corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name
+    name = name.capitalize
```

Comment:
[Optional]
You can use bang methods to directly mutate variables.
```suggestion
    name.capitalize!
```